src/components/RelicModal.jsx

```diff
@@ -331,174 +843,481 @@ export default function RelicModal(props) {
-
-            <Form.Item size="default" name="mainStatValue">
-              <InputNumberStyled controls={false} disabled />
-            </Form.Item>
-          </Flex>
-
-          <HeaderText>Substats</HeaderText>
-
-          <Flex gap={10}>
-            <Form.Item size="default" name="substatType0">
-              <Select
-                showSearch
-                allowClear
-                style={{
-                  width: 200,
-                }}
-                placeholder="Substat"
-                maxTagCount="responsive"
-                options={substatOptions}
-              />
-            </Form.Item>
-
-            <Form.Item size="default" name="substatValue0">
-              <InputNumberStyled controls={false} />
-            </Form.Item>
-          </Flex>
-
-          <Flex gap={10}>
-            <Form.Item size="default" name="substatType1">
-              <Select
-                showSearch
-                allowClear
-                style={{
-                  width: 200,
-                }}
-                placeholder="Substat"
-                maxTagCount="responsive"
-                options={substatOptions}
-              />
-            </Form.Item>
-
-            <Form.Item size="default" name="substatValue1">
-              <InputNumberStyled controls={false} />
-            </Form.Item>
-          </Flex>
-
-          <Flex gap={10}>
-            <Form.Item size="default" name="substatType2">
-              <Select
-                showSearch
-                allowClear
-                style={{
-                  width: 200,
-                }}
-                placeholder="Substat"
-                maxTagCount="responsive"
-                options={substatOptions}
-              />
-            </Form.Item>
-
-            <Form.Item size="default" name="substatValue2">
-              <InputNumberStyled controls={false} />
-            </Form.Item>
-          </Flex>
-
-          <Flex gap={10}>
-            <Form.Item size="default" name="substatType3">
-              <Select
-                showSearch
-                allowClear
-                style={{
-                  width: 200,
-                }}
-                placeholder="Substat"
-                maxTagCount="responsive"
-                options={substatOptions}
-              />
-            </Form.Item>
-
-            <Form.Item size="default" name="substatValue3">
-              <InputNumberStyled controls={false} />
-            </Form.Item>
-          </Flex>
+                  <Form.Item size="default" name="substatValue3">
+                    <InputNumberStyled controls={false} precision={precision3} />
+                  </Form.Item>
+                </Flex>
+              </Flex>
+            </Col>
+            <Col span={8}>
+              <Flex vertical gap={5}>
+                <HeaderText>Upgrade Options</HeaderText>
+                <Form.Item size="default" name="substat0Upgrader">
+                  <Button.Group>
```

Comment:
I think these buttons don't actually have to be in a Form.Item since they're not input values